pepper-apis/src/test/java/org/broadinstitute/ddp/VelocitySketchTest.java

```diff
@@ -0,0 +1,328 @@
+package org.broadinstitute.ddp;
+
+import java.io.Reader;
+import java.io.StringReader;
+import java.io.StringWriter;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+import com.sendgrid.Content;
+import com.sendgrid.Mail;
+import org.apache.commons.io.IOUtils;
+import org.apache.velocity.Template;
+import org.apache.velocity.VelocityContext;
+import org.apache.velocity.app.VelocityEngine;
+import org.broadinstitute.ddp.client.SendGridClient;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class VelocitySketchTest {
+
+    static VelocityEngine velocityEngine = null;
+
+    @BeforeClass
+    public static void setUp() {
+        velocityEngine = new VelocityEngine();
+        // in production, we will probably use more flexible
+        // ways to load macros, such as VelocityEngine.evaluate()
+        Properties velocityProps = new Properties();
+        velocityProps.put("file.resource.loader.path", "/sandbox/ddp-study-server/pepper-apis/src/test/resources");
```

Comment:
test had problems finding the templates until I replaced the path here with my local absolute path for the test/resources directory.